packages/components/src/Button/button.spec.tsx

```diff
@@ -1,31 +1,30 @@
-import { generateSnapshots, prepareStory } from "@chanzuckerberg/story-utils";
+import { generateSnapshots } from "@chanzuckerberg/story-utils";
+import { composeStories } from "@storybook/testing-react";
 import { render, screen } from "@testing-library/react";
 import React from "react";
 import Button from "./button";
 import * as ButtonStoryFile from "./button.stories";
 
+const { Default } = composeStories(ButtonStoryFile);
```

Comment:
Storybook now has a `@storybook/testing-react` for use with testing library. I've retired our own `prepareStory`, in lieu of this.